salt/metalk8s/repo/deployed.sls

```diff
@@ -22,7 +20,7 @@ Deploy repo service object:
         clusterIP: None
         ports:
         - name: http
-          port: {{ repo.port }}
+          port: {{ pillar.metalk8s.endpoints['package-repositories'].ports.http }}
```

Comment:
Similar to the registry service IP before, this should be a fixed value.

Reply:
For the port, what do you mean by fixed value? Hardcoding it or letting it as it was before? (ie configurable via the default.yaml). My idea here is to force the caller of the state to specify the ports in the pillar arg if it is not available via the endpoints yet